platform/ios/src/MGLMapView.mm

```diff
@@ -3649,8 +3649,13 @@ - (void)setCamera:(MGLMapCamera *)camera withDuration:(NSTimeInterval)duration a
     [self cancelTransitions];
 
     self.cameraChangeReasonBitmask |= MGLCameraChangeReasonProgrammatic;
-
-    mbgl::CameraOptions cameraOptions = [self cameraOptionsObjectForAnimatingToCamera:camera edgePadding:edgePadding];
+    
+    mbgl::EdgeInsets padding = MGLEdgeInsetsFromNSEdgeInsets(edgePadding);
+    padding += MGLEdgeInsetsFromNSEdgeInsets(self.contentInset);
```

Comment:
Some callers are already adding the same content inset to the padding:

https://github.com/mapbox/mapbox-gl-native/blob/d00d296cc43d0ce9b2645b7cf8cb48c2d86bac78/platform/ios/src/MGLMapView.mm#L3757

To avoid the redundant change, those callers also need to be updated.